guides/source/association_basics.md

```diff
@@ -1746,7 +1739,7 @@ There may be times when you wish to customize the query used by `has_many`. Such
 
 ```ruby
 class Author < ApplicationRecord
-  has_many :books, -> { where processed: true }
+  has_many :books, -> { where processed: true }, inverse_of: :author
```

Comment:
I think updating all examples is the way to since scopes are only mentioned once in the text, that way when people copy-paste an example they'll get the right thing. We could address this in another way too.

Reply:
`has_many :books, ..., inverse_of: :author` is not always the right association name. At least we have another example `has_many :books, inverse_of: 'writer'` in the guide.

```ruby
class Author < ApplicationRecord
  has_many :books, inverse_of: 'writer'
end
class Book < ApplicationRecord
  belongs_to :writer, class_name: 'Author', foreign_key: 'author_id'
end
```

https://github.com/rails/rails/blame/011f76e57b145b9e5ef1fd520df7d7096cf8896a/guides/source/association_basics.md#L770-L778

Reply:
I agree with @kamipo that those examples don't need to change. They also are giving the impression that if you define an association you already need to define the `inverse_of:`